src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int cargoCapacity;
+    private int numberOfAxles;
+
+    public Truck(int cargoCapacity, int numberOfAxles) {
+        this.cargoCapacity = cargoCapacity;
+        this.numberOfAxles = numberOfAxles;
+    }
+
     public Truck() {
     }
 
+    public int getCargoCapacity() {
+        return cargoCapacity;
+    }
+
+    public Truck setCargoCapacity(int cargoCapacity) {
```

Comment:
Consider adding a comment to the setter method to explain the return value, which is the current instance of `Truck`. This is a fluent interface pattern, and although it's clear for many, explaining it briefly could be helpful for less experienced developers.